samples/convert_file/convert_file.js

```diff
@@ -11,12 +11,12 @@ async function main() {
         body,
     })
     if ('error_code' in response) throw response
-    const { status, id, output } = response
+    const { status, id, outputs } = response
     console.log(`File conversion id: ${id}`)
     console.log(`File conversion status: ${status}`)
     
     // LITTERBOX-START-NON-EDITABLE-SECTION
-    await fsp.writeFile('./output.stl' , output, 'base64')
+    await fsp.writeFile('./output.stl' , outputs['source.stl'], 'base64')
```

Comment:
@jessfraz suggests it'd be better to loop over all outputs, like

```js
for (const [fileName, fileContents] of Object.entries(outputs)) {
  await fsp.writeFile(`./{fileName}` , fileContents, 'base64')
}
```

Reply:
Cool, made the change!

Is there a chance that changing `output.stl` to `source.stl` will break something where those tutorials are used?

Reply:
Like this one https://github.com/KittyCAD/website/blob/586f2f34b65956af9df23602ef4cf4620a87652a/lib/getTabsFromLitterboxPath.test.ts#L45

Reply:
So I think the go example addresses that well (hardcoded output name), so I tried to make the js a lot more like that